handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -859,7 +859,20 @@ public final boolean isServer() {
      * Creates a new {@link SslHandler}.
      * <p>If {@link SslProvider#OPENSSL_REFCNT} is used then the returned {@link SslHandler} will release the engine
      * that is wrapped. If the returned {@link SslHandler} is not inserted into a pipeline then you may leak native
-     * memory!
+     * memory!</p>
+     * <p><b>Beware</b>: the underlying generated {@link SSLEngine} won't have hostname verification enabled by default.
```

Comment:
@slandelle maybe add a link to an explanation what this even is ? (hostname verification )